packages/esm-bed-management-app/src/bed-administration/bed-administration-form.component.tsx

```diff
@@ -275,16 +275,16 @@ const BedAdministrationForm: React.FC<BedAdministrationFormProps> = ({
               />
             )}
           </Stack>
-        </ModalBody>
-        <ModalFooter>
-          <Button onClick={() => onModalChange(false)} kind="secondary">
-            {t('cancel', 'Cancel')}
-          </Button>
-          <Button disabled={!isDirty} type="submit">
-            <span>{t('save', 'Save')}</span>
-          </Button>
-        </ModalFooter>
-      </Form>
+        </Form>
+      </ModalBody>
+      <ModalFooter>
+        <Button onClick={() => onModalChange(false)} kind="secondary">
+          {t('cancel', 'Cancel')}
```

Comment:
Hi @IamMujuziMoses , you could make use of the framework `getCoreTranslation` 
Thanks